patches/minecraft/net/minecraft/client/KeyboardListener.java.patch

```diff
@@ -1,17 +1,16 @@
 --- a/net/minecraft/client/KeyboardListener.java
 +++ b/net/minecraft/client/KeyboardListener.java
-@@ -255,7 +255,9 @@
+@@ -255,7 +255,8 @@
           }
  
           INestedGuiEventHandler inestedguieventhandler = this.field_197972_a.field_71462_r;
 -         if (p_197961_5_ == 1 && (!(this.field_197972_a.field_71462_r instanceof ControlsScreen) || ((ControlsScreen)inestedguieventhandler).field_152177_g <= Util.func_211177_b() - 20L)) {
-+
 +         if ((!(this.field_197972_a.field_71462_r instanceof ControlsScreen) || ((ControlsScreen)inestedguieventhandler).field_152177_g <= Util.func_211177_b() - 20L)) {
```

Comment:
Changing the order here screws up the performance optimization of the simpler condition being at the beginning of the conditionals.